priv/static/Linux.sh

```diff
@@ -0,0 +1,422 @@
+        "phoenix")
+            mix archive.install hex phx_new 1.7.0-rc.2
+            ;;
+        "nodejs")
+            asdf plugin add nodejs https://github.com/asdf-vm/asdf-nodejs.git
+            asdf install nodejs 16.17.0
+            asdf global nodejs 16.17.0
+            ;;
+        "postgresql")
+            sudo apt-get install -y linux-headers-$(uname -r) build-essential libssl-dev libreadline-dev zlib1g-dev libcurl4-openssl-dev uuid-dev
+            asdf plugin add postgres https://github.com/smashedto
+            atoms/asdf-postgres.git
+            asdf install -y postgres latest
+            asdf global postgres latest
+
+            echo -e "${cyan}${bold}Starting the postgres server... ${reset}"
+            pg_ctl start
+            ;;
+        "vim")
+            sudo apt-get install -y vim
+            ;;
+        "chrome")
+            wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+            sudo apt install -y ./google-chrome-stable_current_amd64.deb
+            ;;
+        "chromedriver")
+            npm install -g chromedriver
+            ;;
+        "docker")
+            sudo apt-get update
+            sudo apt-get install -y \
+            ca-certificates \
+            curl \
+            gnupg \
+            lsb-release
+            sudo mkdir -p /etc/apt/keyrings
+            curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+            echo \
+            "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+            $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+            sudo apt-get update
+            sudo apt-get install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
+            ;;
+        *)
+            echo "Invalid name argument on install"
+        ;;
+    esac
+}
+
+function get() {
+    if is_package_exists $1; then
+        echo "$1 is already installed. Skipping..."
+    else
+        echo "Installing $1"
+        if [[ $1 == "homebrew" || $1 == "erlang" ]]; then
+            echo "This might take a while."
+        fi
+        install $1
+    fi
+}
+
+function add_env() {
+    echo ""
+    echo ""
+    echo -e "${cyan}${bold}First we install curl"
+    echo -e "${white}"
+    sleep 1
+    get "curl"
+    echo ""
+
+    echo -e "${cyan}${bold}Now we are going to install zsh (it's a shell variant, required for oh-my-zsh)"
+    sleep 2
+    echo -e "${white}"
+    get "zsh"
+    echo ""
+
+    echo -e "${cyan}${bold}After that we need to add oh-my-zsh terminal helpers"
+    echo -e "${white}"
+    sleep 2 
+    get "oh-my-zsh"
+    echo ""
+
+    echo -e "${cyan}${bold}Now we need wget (command-line tool that makes it possible to download files)"
+    echo -e "${white}"
+    sleep 2
+    get "wget"
+    echo ""
+
+    echo -e "${cyan}${bold}Now we are gonna add Homebrew"
```

Comment:
Standardize the messages. No need for a custom message for each package.
There are only 3 types of messages

```
Installing <tool>...
```
if the tool isn't already installed

```
Installing <tool>...
This might take a while.
```
for Homebrew and Erlang

and
```
<tool> is already installed. Skipping...
```
if the tool is already installed